docs/quick-start/pre-1stflight.md

```diff
@@ -86,6 +86,29 @@ On the 900MHz hardware, a value of -20dBm or thereabouts is a good indication yo
 - Most DIY modules require moving the 0-ohm resistor on the E28 from the PCB antenna side towards the UFL side. A solder bridge will work just fine as well, but make sure it is on the correct pads.
 - Swap out antennas on the receiver and/or the TX module; Most ExpressLRS receiver antennas use IPEX 1/UFL connectors and as long as the frequency tuning of the antenna is appropriate for the frequency you're in, it should work. You can also use old 2.4GHz WiFi Router antennas for your 2.4GHz modules, but avoid those that are dual-band. Also, make sure that the connector on the antenna is the appropriate one (RP-SMA on R9 modules; SMA on most off-the-shelf ExpressLRS modules)
 
+## RC Link Preset
+For flight controllers based on Betaflight, there are 'RC Link Presets' available that configure feedforward smoothing and associated link settings based on your packet rate and usage case. Using no link preset, or using the incorrect one for your packet rate, can result in unwanted disturbances in feedforward, which can affect setpoint tracking.
```

Comment:
> can result in unwanted disturbances in feedforward

would be great if we can be a bit specific. e.g. "can result in jittery or noisy signal"

Reply:
you'll have to add a few words in the wordlist to appease the spellchecker or use different words altogether.